iina/PlayerCore.swift

```diff
@@ -642,10 +651,169 @@ class PlayerCore: NSObject {
     }
   }
 
-  @objc private func reEnableOSDAfterFileLoading() {
+  @objc
+  private func reEnableOSDAfterFileLoading() {
     info.disableOSDForFileLoading = false
   }
 
+  /// Add files in the same folder to playlist.
```

Comment:
Just a question: have you tried running this on a large folder? This function looks like it might not be fast enough if there are too many videos and subtitles.